src/thunderbots/software/test/primitive/stop_primitive.cpp

```diff
@@ -0,0 +1,87 @@
+/**
+ * This file contains the unit tests for the MovePrimitive class
+ */
+
+#include "ai/primitive/stop_primitive.h"
+
+#include <gtest/gtest.h>
+#include <string.h>
+
+TEST(StopPrimTest, construct_with_no_params_test)
+{
+    const std::string stop_prim_name = "Stop Primitive";
+
+    StopPrimitive stop_prim = StopPrimitive(0, true);
+
+    EXPECT_EQ(int(), stop_prim.getRobotId());
```

Comment:
Replace `int()` with `0`. It's a bit more explicit and matches the constructor above